### DISCUSSION TITLE
could you add an option to disable ctrl+mousewheel(touchpad) zoom

### DISCUSSION THREAD

**Discussion by chenzhiwei** [linuxdeepin/developer-center#2929] [Features Request & Ideas | 特性请求 & 头脑风暴]
This is very annoying when writing code from Deepin Terminal in VIM.

`Disable touchpad while typing` feature does not applicable for some of the functional keys, in this case, the `ctrl` key. When you press the `ctrl` key, the touchpad is still in active status.

Konsole added this option in year 2012: https://github.com/KDE/konsole/commit/417af269e556cba6330a736b0b3089a053bff07f

Terminator also added this option, see https://github.com/gnome-terminator/terminator/pull/46

**Comment by ch593030323** [comment]
Thank u for your suggestion. We will consider this function reasonably

**Comment by meiyixiang** [comment]
There are many usage scenarios for the middle mouse button. I will record this problem and make unified plans for the management of the middle mouse button
